espanol/canudos.md

```diff
@@ -121,3 +121,5 @@ Porque en el fondo él sabía que por mucho que hubiese ganado esta batalla,
 
 "El que tenga rabo de paja ni que lo fajen chiquito", 
 dijo el [curandero](curandero/curandero.md)...
+
+[Nadie sabiendo el secreto mejor guardado este epico personaje, "El gran mutante", se atrevio a lo que nadie se imaginó:](Mutantes/Mutantes.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.